test/test_linalg.py

```diff
@@ -1337,17 +1337,16 @@ def run_test_case(input, ord, dim, keepdim):
     def test_norm_fused_type_promotion(self, device, dtype):
         x = torch.randn(10, device=device, dtype=dtype)
 
-        def profile_and_check(fn, x, kwargs, fn_name):
+        def profile_and_check(fn, x, kwargs):
             with torch.profiler.profile(activities=(torch.profiler.ProfilerActivity.CPU,)) as p:
                 fn(x, **kwargs, dtype=torch.float)
             # smoke check that profiler returned some events
-            self.assertTrue(fn_name in map(lambda e: e.name, p.events()))
+            self.assertTrue("aten::linalg_vector_norm" in map(lambda e: e.name, p.events()))
             # test that there was no explicit copy
             self.assertFalse("aten::to" in map(lambda e: e.name, p.events()))
```

Comment:
Maybe also change while we are here.

`self.assertFalse("aten::to" in (e.name for e in p.events()))`